config/plugins/visualizations/chiraviz/static/js/rna-viz.js

```diff
@@ -9,8 +9,7 @@ in multiple plots, comic alignments
 var RNAInteractionViewer = (function(riv) {
   riv.configObject = null;
   riv.model = null;
-  riv.showRecords = 50;
-  riv.nIncrement = 50;
+  riv.nIncrement = 25;
```

Comment:
`riv.showRecords` should really be removed?

Reply:
Yes, it's not being used.